readthedocs/templates/core/email/base.html

```diff
@@ -63,7 +63,7 @@
           <tr>
             <td class="header">
               {% block header %}
-              <img src="https://assets.readthedocs.org/static/images/email-header.png" width="100%" height="150" />
+              <img src="https://assets.readthedocs.org/static/images/email-header.png" width="100%" height="150">
```

Comment:
The original is fine here, and I still close void tags this way for consistency and readability. Closing void tags like this is perfectly fine by html5 specs, and it is at least explicit about the end of void tags.

Reply:
But isn't this unnecessary? I had tested it several times, it also works without the /!

Reply:
Either is fine as per html5 spec, closing of void tags is an xhtml specification. Either way, this change is not necessary. Closing of void tags is a code preference that we still use in our html as there is a readability improvement from explicitly closing all tags.

Reply:
Fixed, see fb03d3a